lib/Jsrt/ChakraDebug.h

```diff
@@ -538,7 +538,6 @@
             _In_ unsigned int objectHandle,
             _Out_ JsValueRef *handleObject);
 
-#ifdef _WIN32
```

Comment:
I had done the underlying work for JsrtDebuggerStackFrame::Evaluate to take char16* and length. Please create a utf8 version and have ch use that?